### ISSUE TITLE
fails to rewrap a markdown file if there is a commented-out code block

### ISSUE THREAD

**Issue by paul-white-intel** [stkb/Rewrap#288]
I've attached a markdown file that exhibits this bug. The paragraphs after a commented-out code block fail to rewrap, but the paragraphs _before_ are fine.
[test.md](https://github.com/stkb/Rewrap/files/7247558/test.md)

```

> this paragraph will wrap correctly 
> 
> <!--
> ```
> some code
> ```
> -->
> 
> This paragraph will not

```

**Comment by stkb** [comment]
Thanks for the test case! [This will be fixed in the next version](https://github.com/stkb/Rewrap/releases/tag/beta).

**Comment by paul-white-intel** [comment]
Thanks!

**Comment by stkb** [comment]
Fixed in v1.16.0
